Makefile

```diff
@@ -324,27 +355,48 @@ gen-cli-docs: ${KOPS} # Regenerate CLI docs
 	KOPS_FEATURE_FLAGS= \
 	${KOPS} genhelpdocs --out docs/cli
 
-.PHONY: push
-# Will always push a linux-based build up to the server
-push: crossbuild-nodeup
+.PHONY: push-amd64
+push-amd64: crossbuild-nodeup-amd64
 	scp -C .build/dist/linux/amd64/nodeup  ${TARGET}:/tmp/
 
-.PHONY: push-gce-dry
-push-gce-dry: push
+.PHONY: push-arm64
+push-arm64: crossbuild-nodeup-arm64
+	scp -C .build/dist/linux/arm64/nodeup  ${TARGET}:/tmp/
+
+.PHONY: push-gce-dry-amd64
+push-gce-dry-amd64: push-amd64
+	ssh ${TARGET} sudo /tmp/nodeup --conf=metadata://gce/instance/attributes/config --dryrun --v=8
+
+.PHONY: push-gce-dry-arm64
+push-gce-dry-arm64: push-arm64
 	ssh ${TARGET} sudo /tmp/nodeup --conf=metadata://gce/instance/attributes/config --dryrun --v=8
 
-.PHONY: push-gce-dry
-push-aws-dry: push
+.PHONY: push-aws-dry-amd64
+push-aws-dry-amd64: push-amd64
 	ssh ${TARGET} sudo /tmp/nodeup --conf=/opt/kops/conf/kube_env.yaml --dryrun --v=8
 
-.PHONY: push-gce-run
-push-gce-run: push
+.PHONY: push-aws-dry-arm64
+push-aws-dry-arm64: push-arm64
+	ssh ${TARGET} sudo /tmp/nodeup --conf=/opt/kops/conf/kube_env.yaml --dryrun --v=8
+
+.PHONY: push-gce-run-amd64
+push-gce-run-amd64: push-amd64
 	ssh ${TARGET} sudo cp /tmp/nodeup /var/lib/toolbox/kubernetes-install/nodeup
 	ssh ${TARGET} sudo /var/lib/toolbox/kubernetes-install/nodeup --conf=/var/lib/toolbox/kubernetes-install/kube_env.yaml --v=8
 
+.PHONY: push-gce-run-arm64
+push-gce-run-arm64: push-arm64
+	ssh ${TARGET} sudo cp /tmp/nodeup /var/lib/toolbox/kubernetes-install/nodeup
+	ssh ${TARGET} sudo /var/lib/toolbox/kubernetes-install/nodeup --conf=/var/lib/toolbox/kubernetes-install/kube_env.yaml --v=8
+
+# -t is for CentOS http://unix.stackexchange.com/questions/122616/why-do-i-need-a-tty-to-run-sudo-if-i-can-sudo-without-a-password
+.PHONY: push-aws-run-amd64
+push-aws-run-amd64: push-amd64
+	ssh -t ${TARGET} sudo /tmp/nodeup --conf=/opt/kops/conf/kube_env.yaml --v=8
+
 # -t is for CentOS http://unix.stackexchange.com/questions/122616/why-do-i-need-a-tty-to-run-sudo-if-i-can-sudo-without-a-password
-.PHONY: push-aws-run
```

Comment:
I think there are a few references to these make targets in the docs directory that should get updated.

Reply:
Thanks for noticing this. Done.